### PR TITLE
Docs: Update instructions for app management.

### LINKED ISSUE #152: Use AiiDAlab: Using App Store
Add information on how to use App Store for installing, updating, uninstalling applications.

### PR DESCRIPTION
**Pull request by csadorf**
Closes #152.

### CODE REVIEW

**Comment by csadorf** [comment]
@mbercx The updated section starts here: https://aiidalab--260.org.readthedocs.build/en/260/usage/index.html#aiidalab-home-page

File: docs/source/usage/index.rst [hunk 1]
```diff
@@ -73,15 +73,209 @@ When opening AiiDAlab you see a list of installed applications.
...
+       :container: container-lg pb-3
+       :column: col-lg-12 p-2
+
+       **Step 1: Find the app you would like to uninstall on the start page.**
+
+       On the home app start page, simply look for the app you would like to uninstall.
+
+       .. image:: ../_static/app-management-start-page.png
+
+       Click on the **Manage App** button to open the app manager:
```
Comment (mbercx) [inline docs/source/usage/index.rst:252]: Found the colon to look a little strange here.

```suggestion
       Click on the **Manage App** button to open the app manager.
```

File: docs/source/usage/index.rst [hunk 2]
```diff
@@ -73,15 +73,209 @@ When opening AiiDAlab you see a list of installed applications.
...
+       ~$ aiidalab install quantum-espresso==v22.01.0
+
+.. _app-store:uninstall:
+
+Uninstall an app
+================
+
+Uninstalling an app works similar to upgrading or downgrading an app via the **Manage App** page or on the Terminal.
+
+.. tabbed:: App Store
```
Comment (mbercx) [inline docs/source/usage/index.rst:240]: Although it's part of the App store, perhaps "App manager" is more clear for users here?

```suggestion
.. tabbed:: App manager
```

File: docs/source/usage/index.rst [hunk 3]
```diff
@@ -73,15 +73,209 @@ When opening AiiDAlab you see a list of installed applications.
...
+
+       By default, the app will be upgraded to the latest available version, howevever you can alternatively select any available version, including a version that is lower than the currently installed one.
+
+.. tabbed:: Terminal
+
+    Within the :fa:`terminal` Terminal, execute the following command to upgrade:
+
+    .. code-block:: console
+
+       ~$ aiidalab install <app-name>
```
Comment (mbercx) [inline docs/source/usage/index.rst:222]:
```suggestion
       $ aiidalab install <app-name>
```

File: docs/source/usage/index.rst [hunk 4]
```diff
@@ -73,15 +73,209 @@ When opening AiiDAlab you see a list of installed applications.
...
+
+    This will install the most recent version of an app, regardless of whether it is already installed or not.
+    You will be prompted to confirm the operation.
+
+    You can install a specific version, by using standard `PEP 440 version specifiers`_, for example:
+
+    .. code-block:: console
+
+       ~$ aiidalab install quantum-espresso==v22.01.0
```
Comment (mbercx) [inline docs/source/usage/index.rst:231]:
```suggestion
       $ aiidalab install quantum-espresso==v22.01.0
```

File: docs/source/usage/index.rst [hunk 5]
```diff
@@ -73,15 +73,209 @@ When opening AiiDAlab you see a list of installed applications.
...
+          That warning indicates that there are local modifications to the app source code.
+          You can safely ignore this warning and click on the "Ignore" check box in case that you are sure that any local modifications are safe to delete.
+
+.. tabbed:: Terminal
+
+    Within the :fa:`terminal` Terminal, execute the following command to uninstall an app:
+
+    .. code-block:: console
+
+       ~$ aiidalab uninstall <app-name>
```
Comment (mbercx) [inline docs/source/usage/index.rst:276]:
```suggestion
       $ aiidalab uninstall <app-name>
```

**Review by mbercx — APPROVED** [review]
The documentation section looks great, thanks @csadorf! I've just left a few minor nitpicks.

**Comment by csadorf** [comment]
> The documentation section looks great, thanks @csadorf! I've just left a few minor nitpicks.

Thanks for the review and the very helpful suggestions!

**Pull request merged**